4-Deployment/1-deploy-storage/SPA/src/authConfig.js

```diff
@@ -62,12 +62,8 @@ export const loginRequest = {
  * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/resources-and-scopes.md
  */
 export const protectedResources = {
-    graphMe: {
-        endpoint: "https://graph.microsoft.com/v1.0/me",
-        scopes: ["User.Read"],
-    },
     apiHello: {
-        endpoint: "Enter_the_Web_Api_Uri_Here", // e.g. https://my-web-api.azurewebsites.net/api
-        scopes: ["Enter_the_Web_Api_Scope_Here"], // e.g. api://xxxxxx/access_as_user
+        endpoint: 'http://localhost:5000/hello', // e.g. https://my-web-api.azurewebsites.net/hello
```

Comment:
```suggestion
        endpoint: 'http://localhost:5000/hello', // e.g. https://my-web-api.azurewebsites.net/hello
```

Can you emphasize (here or in README) that this needs to be updated with the deployed app uri?

Reply:
It's already in the SPA [deployment steps](https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial/tree/update-storage-deployment-sample/4-Deployment/1-deploy-storage#step-3-update-the-client-apps-configuration-files): 
1-Find the key protectedResources.apiHello.endpoint and replace the existing value with your published web API's endpoint, e.g. https://node-webapi-1.azurewebsites.net/hello